### PR TITLE
'evaluation' correctional on editorial page

### PR DESCRIPTION
**Pull request by ksmartin301**
Screenshot 2024-07-11 at 5 59 09 AM
['evaluation' correctional on editorial page](https://github.com/Cyber4All/clark-client/pull/1769/commits/92529469ddf38095766034686d5602a789bbc542)

### CODE REVIEW

**Review by curlyLasagna — APPROVED** [review]

File: src/app/cube/content-pages/editorial-process/copy.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export const sections = {
 						question: 'Why must my curriculum be evaluated?',
 						answer: 'Since all released curricula fall under Creative Commons, all types of curricula must be evaluated' +
                         ' to ensure educational quality, adaptability, and accountability in the learning process. Any curriculum to be' +
-                        ' released on CLARK must comply with all of the characteristics mentioned in our evaluatuion procress. '
+                        ' released on CLARK must comply with all of the characteristics mentioned in our evaluation procress. '
```
Comment (pzalep1) [inline src/app/cube/content-pages/editorial-process/copy.ts:12]: process too, not procress

**Review by pzalep1 — CHANGES_REQUESTED** [review]

File: src/app/cube/content-pages/editorial-process/copy.ts (hunk 1 above)
Comment (mfranz1) [inline src/app/cube/content-pages/editorial-process/copy.ts:12]:
```suggestion
                        ' released on CLARK must comply with all of the characteristics mentioned in our evaluation process. '
```

**Review by mfranz1 — CHANGES_REQUESTED** [review]

**Review by johngonz0 — APPROVED** [review]
Looks good, just page and mikes comments.

**Comment by stef359** [comment]
Mike and Paige's comments and it should be good

**Review by stef359 — APPROVED** [review]

**Review by shanioffe — APPROVED** [review]
Just the spell checks and you're good to go

**Review by mfranz1 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Pull request merged**